analyses/cell-type-wilms-tumor-14/run_cell-type-wilms-14.sh

```diff
@@ -31,22 +32,26 @@ meta_path="${data_dir}/single_cell_metadata.tsv"
 step_name="00_preprocess_reference"
 scratch_dir_step="${scratch_dir}/${step_name}" && mkdir -p ${scratch_dir_step}
 
+
+
 # Download and process reference data
 ref_h5ad="${scratch_dir_step}/Fetal_full_v3.h5ad"
 ref_seurat="${scratch_dir_step}/kidneyatlas.rdsSeurat"
 ref_seurat_sct="${scratch_dir_step}/kidneyatlas_SCT.rdsSeurat"
 
+echo "Downloading reference data"
 if [[ ! -e ${ref_h5ad} ]]; then
     ref_url="https://cellgeni.cog.sanger.ac.uk/kidneycellatlas/Fetal_full_v3.h5ad"
     curl -s -o ${ref_h5ad} ${ref_url}
```

Comment:
The error 6 seems to be coming here, so I am going to add a few options to try to make this more reliable and add the error report
```suggestion
    curl -s --show-error -L -o ${ref_h5ad} ${ref_url}
```

Reply:
I wonder if this would be a more stable link?
https://datasets.cellxgene.cziscience.com/fcb6225a-b329-4ac9-8a3f-559ca9bac50e.h5ad

From https://cellxgene.cziscience.com/collections/120e86b4-1195-48c5-845b-b98054105eec, which should be [the same](https://www.kidneycellatlas.org/). This is where we download from for wilms-06, which I'm realizing probably also needs to be changed to an anndata link...

Reply:
Unfortunately, these are not the same file. I can't tell you what the difference is, but they are not the same size, so I don't want to substitute for the purposes of this PR

This seems to be a DNS issue; I never have trouble downloading the file locally.